client/client_test.go

```diff
@@ -55,7 +57,7 @@ func TestInvalidAuth0DomainValue(t *testing.T) {
 
 func TestServerDown(t *testing.T) {
 	test := func(keycloakProvider bool) {
-		http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
+		// http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true}
```

Comment:
The `http.DefaultTransport` is used to configure the `http.DefaultClient`, since we are now using a custom client to set the `TLSClientConfig`, we should use the `skipTLSVerify` set to true on these tests to skip the verification, and then remove the `http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true}` configuration.

Reply:
Perfect, I will remove the ` http.DefaultTransport.(*http.Transport).TLSClientConfig = &tls.Config{InsecureSkipVerify: true}`